lib/membrane_h264_plugin/parser/decoder_configuration_record.ex

```diff
@@ -0,0 +1,56 @@
+defmodule Membrane.H264.Parser.DecoderConfigurationRecord do
+  @moduledoc """
+  Utility functions for parsing AVC Configuration Record
```

Comment:
Let's add here some reference that this record is defined in MPEG-4 part 15 (ISO/IEC
14496-15)